en/docs/quick-starts/android-sample.md

```diff
@@ -14,13 +14,11 @@ using the [PKCE extension](https://tools.ietf.org/html/rfc7636)
 
 ## Set up the sample
 
-1.  Download the [android sample](https://github.com/wso2-extensions/identity-samples-android/archive/master.zip) and unzip it.
+Clone the sample from this [sample repository](https://github.com/asgardeo/asgardeo-android-oidc-sdk).
 
-2.  Else, clone the sample from this [sample repository](https://github.com/wso2-extensions/identity-samples-android.git).
-
-    ```
-    git clone https://github.com/wso2-extensions/identity-samples-android.git
-    ```
+```
+git clone https://github.com/asgardeo/asgardeo-android-oidc-sdk
+```
```

Comment:
is this content: https://github.com/wso2/docs-is/pull/2463/files#diff-3422f5a28bd77364dd04b3fd14164174b5f8053c5ecc82b45a8a0b0b5d9dcf9eR23-L107 valid for the new sample?

Reply:
We don't have a built sample. We need to get a clone of the repo to run the android sample.

Reply:
Yes, these configurations are specific to the sample. Please refer to the instructions in https://github.com/asgardeo/asgardeo-android-oidc-sdk#configuring-the-sample.